push-sample-app/src/main/java/example/okta/android/sample/errors/BiometricError.kt

```diff
@@ -16,5 +16,7 @@ package example.okta.android.sample.errors
 
 sealed class BiometricError(errorCode: Int, errString: CharSequence) : Exception("errorCode $errorCode errString $errString") {
     class UserCancel(errorCode: Int, errString: CharSequence) : BiometricError(errorCode, errString)
+    class UvTemporaryUnavailable(errorCode: Int, errString: CharSequence) : BiometricError(errorCode, errString)
```

Comment:
can drop the "Uv" from these names because the sealed class is already BiometricError